node_modules/oae-core/createcollabdoc/js/createcollabdoc.js

```diff
@@ -110,8 +110,8 @@ define(['jquery', 'oae.core'], function ($, oae) {
                 // Update visibility for document
                 visibility = data.visibility;
 
-                managers = data.shared;
-                folders = data.folders;
+                managers = _.pluck(data.selectedPrincipalItems, 'shareId');
```

Comment:
No because we need more than just the `shareId` for principals (we need the principal `id` sometimes, which is not the same as `shareId`), and it's better to just give all the information we have in a consistent model between folder and principals than give some selected items as ids and some selected items as more complex objects